docs/src/app/(public)/(content)/react/utils/use-render/demos/data-attributes/css-modules/index.tsx

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable react/button-has-type, jsx-a11y/control-has-associated-label */
```

Comment:
`render ?? <button />` requires to disable these rules, or add the required props to `<button />` but it's also weird.

Reply:
Why not moving the type and aria-label from the props here?

Reply:
Maybe then list the file it the eslint ignore config? I don't want people to have to see it in the demo.

Reply:
Done in https://github.com/mui/base-ui/pull/1418/commits/ab396658da035c6564374797c397480976051ff3. Not sure if the preference would be to ignore individual demos instead of the whole /react folder for the `jsx-a11y/control-has-associated-label` rule.